docs/self-managed/zeebe-deployment/configuration/logging.md

```diff
@@ -3,65 +3,53 @@ id: logging
-  </Properties>
-
-  <Appenders>
-    <Console name="Console" target="SYSTEM_OUT">
-      <PatternLayout
-        pattern="${log.pattern}"/>
-    </Console>
-
-    <Console name="Stackdriver" target="SYSTEM_OUT">
-      <StackdriverLayout serviceName="${log.stackdriver.serviceName}"
-        serviceVersion="${log.stackdriver.serviceVersion}" />
-    </Console>
-
-    <RollingFile name="RollingFile" fileName="${log.path}/zeebe.log"
-      filePattern="${log.path}/zeebe-%d{yyyy-MM-dd}-%i.log.gz">
-      <PatternLayout>
-        <Pattern>${log.pattern}</Pattern>
-      </PatternLayout>
-      <Policies>
-        <TimeBasedTriggeringPolicy/>
-        <SizeBasedTriggeringPolicy size="250 MB"/>
-      </Policies>
-    </RollingFile>
-  </Appenders>
-
-  <Loggers>
-    <Logger name="io.camunda.zeebe" level="${env:ZEEBE_LOG_LEVEL:-info}"/>
-
-    <Logger name="io.atomix" level="${env:ATOMIX_LOG_LEVEL:-warn}"/>
-
-    <Root level="info">
-      <AppenderRef ref="RollingFile"/>
-
-      <!-- remove to disable console logging -->
-      <AppenderRef ref="${env:ZEEBE_LOG_APPENDER:-Console}"/>
-    </Root>
-  </Loggers>
-
-</Configuration>
-```
+[You can find the default log4j2.xml used by the application here](https://github.com/camunda/camunda/blob/main/dist/src/main/config/log4j2.xml).
+
+It will configure the default [log level](https://logging.apache.org/log4j/2.x/manual/customloglevels.html) to be `WARN` for all except for the
+following as `INFO`:
+
+- Camunda 8 (anything under `io.camunda` and `io.atomix`)
+- Spring and Spring Boot (anything under `org.springframework`)
+
+:::note
+[See our documentation on log levels to understand what the various log levels mean.](../../operational-guides/monitoring/log-levels.md)
+:::
+
+You can control the log level for the orchestration cluster via the `CAMUNDA_LOG_LEVEL` environment variable, e.g. `CAMUNDA_LOG_LEVEL=DEBUG`, or
+`CAMUNDA_LOG_LEVEL=ERROR`.
+
+:::note
+You can further use the following environment variables to set the log level for certain components of the application, but we recommend using the general
+`CAMUNDA_LOG_LEVEL` unless you really know what you're doing.
+
+- `ZEEBE_LOG_LEVEL`: will set the level for anything under `io.camunda.zeebe`, i.e. Zeebe related.
+- `ATOMIX_LOG_LEVEL`: will set the level for anything clustering or raft related.
+- `OPTIMIZE_LOG_LEVEL`: will set the level for anything under `io.camunda.optimize`.
+- `ES_LOG_LEVEL`: will set the level for anything under `org.elasticsearch`.
+  :::
+
+Additionally, it will configure three possible [appenders](https://logging.apache.org/log4j/2.x/manual/appenders.html) (aka outputs):
+
+- `RollingFile`: [A rolling file appender](https://logging.apache.org/log4j/2.x/manual/appenders/rolling-file.html) which prints out to a file
+  (by default, `logs/zeebe.log` relative from the distribution root). After a day, or once that file reaches 250MB, the file is "rolled" into a
```

Comment:
🔧 I would remove this, as this totally depends on the load, log level, connectivity issues, etc.
```suggestion
  (by default, `logs/zeebe.log` relative from the distribution root). Once that file reaches 250MB, the file is "rolled" into a
```

Reply:
We have the `<TimeBasedTriggeringPolicy>` configured, so it will also roll it after a day if it hasn't reached 250MB.

> The TimeBasedTriggeringPolicy causes a rollover, when the smallest time unit in filePattern changes value. So the rollover can occur at the end of a month, week, at midnight, end of the hour, etc.

Since our smallest unit is the day, then after a day (or so I understand) it will rollover.

Reply:
Ah ok I missed that sorry